src/app/pages/MediaArticlePage/MediaArticlePage.tsx

```diff
@@ -53,11 +58,26 @@ import SecondaryColumn from './SecondaryColumn';
 
 import styles from './MediaArticlePage.styles';
 
-const MediaArticlePage = ({ pageData }) => {
+type MediaArticlePageProps = {
+  pageData: typeof articleDataPropTypes;
+};
+
+type ComponentToRenderProps = {
+  blocks: OptimoBlock[];
+};
+
+type TimestampProps = {
+  firstPublished: number;
+  lastPublished: number;
+  popOut: boolean;
+  minutesTolerance?: number;
+  className: string;
+};
```

Comment:
Is it worth extracting these types into a separate `types.ts` file within the src/app/pages/MediaArticlePage/ directory?

Reply:
Yea good shout